data/json/monsters/zed_lieutenant.json

```diff
@@ -431,5 +468,22 @@
         "force": true
       }
     ]
+  },
+  {
+    "type": "snippet",
+    "category": "Shadow_Warnings_Snippets_early",
+    "text": [
+      "Even out here you can still see the burning cities lighting up the horizon.  What scares you most is not the gunshots you sometimes hear in the far distance, but the idea of what people keep shooting at.  Tonight you hear automatic gunfire winding down into single shots.\n\nJust what could make them shoot that much, all the way out here?",
+      "You suddenly realize this area seems almost devoid of life.  The few bits of blackened grass you can see are barely recognizable as grass, even discounting the night.  Sure the zombies might be tearing people apart, but what happened to the grass?  What, is the grass coming to life to eat people too?\nYou snort at the idea, but it doesn't feel safe out here."
```

Comment:
Gotta admit, these snippets are definitely better written than the warning I came up with.